.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,31 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+on:
+  - pull_request
+  - push
+
+name: "Continuous Integration"
+
+jobs:
+  continuous-integration:
+    name: "Continuous Integration"
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-binary:
+          - php7.3
+
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@master
+
+      - name: "Install dependencies with composer"
+        run: $(which composer) install
```

Comment:
@localheinz would you try throwing a `--ignore-platform-reqs` here to see if this gets us past this issue of misisng ext/intl?

Reply:
Not sure about that, to be honest.

Let's see what we need `ext/intl` for!

Reply:
Running

```
$ composer remove ext-intl
```

followed by

```
$ composer require-checker
```

in #727 yields

```
> composer-require-checker --config-file=composer-require-checker.json
ComposerRequireChecker 2.1.20-250-g14887fc-dev
The following unknown symbols were found:
+----------------+--------------------+
| unknown symbol | guessed dependency |
+----------------+--------------------+
| Transliterator | ext-intl           |
+----------------+--------------------+
Script composer-require-checker --config-file=composer-require-checker.json handling the require-checker event returned with error code 1
```

so we actually need `ext/intl`.

Personally, I'd prefer not to run `composer` with the `--ignore-platform-reqs` options in CI.

Perhaps we can bug @nat with updating the images?

Reply:
Also see https://github.community/t5/GitHub-Actions/Have-PHP-Pecl-installed-by-default/m-p/33371#M1480.

Reply:
Also see https://github.com/actions/virtual-environments/issues/3.